molecule/default/molecule.yml

```diff
@@ -3,29 +3,37 @@
 dependency:
   name: galaxy
 driver:
-  name: docker
-lint:
-  name: yamllint
-  options:
-    config-file: .yamllint.yml
+  name: ${LSR_MOLECULE_DRIVER:-docker}
+lint: |
+  set -eux
+  ansible-lint -v
 platforms:
   - name: centos-6
     image: docker.io/linuxsystemroles/centos-6
```

Comment:
Just a question: Why we not use `registry.centos.org` also for CentOS 6 image?

Reply:
> Just a question: Why we not use `registry.centos.org` also for CentOS 6 image?

I don't know if there is a stock centos6 image with systemd enabled (or for whatever reason we have a custom centos6 image).  That's why I stuck with the linuxsystemroles/centos-6 image.  If we can use a stock image, that would be better.

Reply:
CentOS 6 is too old for systemd AFAIK.

Reply:
> CentOS 6 is too old for systemd AFAIK.

Then I'm not sure why we need a custom image for centos6 - does anyone know?